Templates/DefaultComponent/Template/Include/${GemName}/${Name}Interface.h

```diff
@@ -0,0 +1,33 @@
+// {BEGIN_LICENSE}
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+// {END_LICENSE}
+
+#pragma once
+
+#include <AzCore/Component/ComponentBus.h>
+
+namespace ${GemName}
+{
+    class ${SanitizedCppName}Requests
```

Comment:
Component buses are going to become extremely problematic as we increasingly try to multithread the engine. In fact, I'm fairly certain we're going to have to painfully remove all component buses from the engine to seriously move forward on our performance story.

We should consider carefully if we want component buses to be a default feature of our component template. Multiplayer doesn't use any component buses as an example, it turns out they're not really required for the vast majority of gameplay code.